iib/web/models.py

```diff
@@ -137,6 +137,33 @@ def get_or_create(cls, pull_specification):
         return image
 
 
+class Operator(db.Model):
+    """
+    An operator that has been handled by IIB.
+    """
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String, nullable=False, unique=True)
```

Comment:
There should be an index on this column since we will often filter on it when performing a `get_or_create`.

I realize I forgot to add one on `Image.pull_specification` though. Could you please add one to to it in a separate commit?